fontc/src/lib.rs

```diff
@@ -799,21 +839,26 @@ mod tests {
         let build_dir = temp_dir.path();
 
         let result = compile(Args::for_test(build_dir, "wght_var.designspace"));
+        let mut completed = result.work_completed.iter().cloned().collect::<Vec<_>>();
+        completed.sort();
         assert_eq!(
-            HashSet::from([
-                FeWorkIdentifier::InitStaticMetadata.into(),
+            vec![
+                AnyWorkId::Fe(FeWorkIdentifier::InitStaticMetadata),
                 FeWorkIdentifier::GlobalMetrics.into(),
-                FeWorkIdentifier::FinalizeStaticMetadata.into(),
-                FeWorkIdentifier::Features.into(),
                 FeWorkIdentifier::Glyph("bar".into()).into(),
                 FeWorkIdentifier::Glyph("plus".into()).into(),
+                FeWorkIdentifier::FinalizeStaticMetadata.into(),
+                FeWorkIdentifier::Features.into(),
                 BeWorkIdentifier::Features.into(),
-                BeWorkIdentifier::Glyph("bar".into()).into(),
-                BeWorkIdentifier::Glyph("plus".into()).into(),
                 BeWorkIdentifier::Avar.into(),
                 BeWorkIdentifier::Cmap.into(),
                 BeWorkIdentifier::Fvar.into(),
                 BeWorkIdentifier::Glyf.into(),
+                BeWorkIdentifier::GlyfFragment("bar".into()).into(),
+                BeWorkIdentifier::GlyfFragment("plus".into()).into(),
+                BeWorkIdentifier::Gvar.into(),
+                BeWorkIdentifier::GvarFragment("bar".into()).into(),
+                BeWorkIdentifier::GvarFragment("plus".into()).into(),
```

Comment:
why are these duplicated here?

Reply:
Do you mean Glyf  vs Gvar fragments? Or am I misinterpreting what's beng duplicated?

Reply:
oh sorry, I misread.